uniffi_bindgen/src/bindings/kotlin/templates/RustBufferHelpers.kt

```diff
@@ -53,39 +115,104 @@ fun Long.Companion.read(buf: ByteBuffer): Long {
     return buf.getLong()
 }
 
-// Unsigned types
+fun Long.lower(): Long {
+    return this
+}
+
+fun Long.write(buf: RustBufferBuilder) {
+    buf.putLong(this)
+}
+
+{% when Type::UInt8 -%}
+
+@ExperimentalUnsignedTypes
 fun UByte.Companion.lift(v: Byte): UByte {
     return v.toUByte()
 }
 
+@ExperimentalUnsignedTypes
 fun UByte.Companion.read(buf: ByteBuffer): UByte {
     return UByte.lift(buf.get())
 }
 
+@ExperimentalUnsignedTypes
+fun UByte.lower(): Byte {
+    return this.toByte()
+}
+
+@ExperimentalUnsignedTypes
+fun UByte.write(buf: RustBufferBuilder) {
+    buf.putByte(this.toByte())
+}
+
+{% when Type::UInt16 -%}
+
+@ExperimentalUnsignedTypes
 fun UShort.Companion.lift(v: Short): UShort {
     return v.toUShort()
 }
 
+@ExperimentalUnsignedTypes
 fun UShort.Companion.read(buf: ByteBuffer): UShort {
     return UShort.lift(buf.getShort())
 }
 
+@ExperimentalUnsignedTypes
+fun UShort.lower(): Short {
```

Comment:
This might be an ideal time to change the visibility of these methods to `private` or `internal`.

Reply:
Oof, yes, it had somehow slipped my mind that Kotlin defaults to public visibility. I'll update these to `internal`.